README.md

```diff
@@ -1,13 +1,92 @@
+![Build Status](https://app.bitrise.io/app/7bb2e98294cf2d27/status.svg?token=ISfEDlJWKck83OBeX4XaEQ&branch=master)
+
+# RSDKUtils
 ## Rakuten's SDK Team internal utilities module
 
-#### Test SPM
-**WARNING:** Use RSDKUtils.xcworkspace to test this package. The following method is not available because of bug https://bugs.swift.org/browse/SR-13773
+RSDKUtils module contains useful utilities, extensions, and common classes used in SDK development.
+
+RSDKUtils consists of 4 sub-modules:
+* *Main* - Contains the most common utilities for various purposes like networking, data storage, and standard class extensions.
+* *TestHelpers* - Contains useful mocks and XCTest extensions.
+* *Nimble* - Adds additional Nimble expectations like `toAfterTimeout`.
+* *RLogger* - A tool for managing your log messages.
+
+This module supports iOS 11.0 and above. It has been tested with iOS 11.1 and above.
+
+# **How to install**
+
+## Installing with CocoaPods
+To use the module in its basic configuration your `Podfile` should contain:
+
+```ruby
+# Defined also as 'RSDKUtils/Main'
+pod 'RSDKUtils', :git => 'https://github.com/rakutentech/ios-sdkutils.git'
+```
+
+To use other functionalities, add their respective subspec to your `Podfile`:
+```ruby
+pod 'RSDKUtils/TestHelpers', :git => 'https://github.com/rakutentech/ios-sdkutils.git'
+pod 'RSDKUtils/Nimble', :git => 'https://github.com/rakutentech/ios-sdkutils.git'
+pod 'RSDKUtils/RLogger', :git => 'https://github.com/rakutentech/ios-sdkutils.git'
+```
+
+Run `pod install` to install the module.<br>
+More information on installing pods: [https://guides.cocoapods.org/using/getting-started.html](https://guides.cocoapods.org/using/getting-started.html)
+
+## Installing with Swift Package Manager
+Open your project settings in Xcode and add a new package in 'Swift Packages' tab:
+* Repository URL: `https://github.com/rakutentech/ios-sdkutils.git`
+* Version settings: branch `master` or >= 2.0.0
```

Comment:
I thought for SPM we had to use a range cos it doesn't support >= ?

Reply:
Yeah. ">=" is confusing. I'll fix that